src/special-types-and-traits.md

```diff
@@ -0,0 +1,127 @@
+# Special types and traits
+
+Certain types and traits that exist in [the standard library] are known to the
+Rust compiler (usually by [lang items]). This chapter documents the special
```

Comment:
Language items are unstable, and per #119, we don't want to document unstable things, we shouldn't link or talk about them.